web/app/view/map/BaseMap.js

```diff
@@ -38,14 +38,21 @@ Ext.define('Traccar.view.map.BaseMap', {
         bingKey = server.get('bingKey');
 
         switch (type) {
-            case 'custom':
+            case 'customXyzFormat':
```

Comment:
Why are you changing existing layout key? It will break everyone's configuration.

Reply:
Oh, you're right, I didn't think about that, it will break everyone's configuration. I'll do it the way you suggest bellow. Thanks...